interop/test/ping.spec.ts

```diff
@@ -140,7 +140,7 @@ describe('ping test', () => {
           options.streamMuxers = [mplex()]
           break
         case 'yamux':
-          options.streamMuxers = [yamux()]
+          options.streamMuxers = [mplex()]
```

Comment:
I think this crept in

Reply:
No, this breaks the Stream interface, which`@chainsafe/yamux` has a dependency on.  `@libp2p/mplex` here is updated to the new version but `@libp2p/interface` will need a release that `@chainsafe/yamux` can be upgraded to.

Groan.

I guess I could include it in this PR as `@libp2p/yamux` or something then we can swap out `@libp2p/yamux` for `@chainsafe/yamux` after the release?

Reply:
Oh darn. I guess this is the issue with relying on packages outside the monorepo when doing a breaking change.

At least you could make the diff smaller with somehting like:
```ts
// TODO replace this import once @chainsafe/libp2p-yamux has been updated
import { mlex as yamux } from '@libp2p/mplex'
```

Reply:
I've temporarily added yamux to the monorepo but with `"private": true"` in the `package.json`, this should let us build and publish the other packages without trying to publish yamux, we can then open a PR to yamux afterwards with the changes and remove it from here.